node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -560,6 +562,7 @@ func (o GetDmsPathHelperGeneric) WaitForDmToExist(volumeUuid string, volumeNguid
 	formatTemplate := strings.Join([]string{"%d", "%w"}, mpathdSeparator)
 	args := []string{"show", "maps", "raw", "format", "\"", formatTemplate, "\""}
 	var err error
+	logger.Debugf("Waiting for dm to exist, executing command : {%v} with args : {%v}.", multipathdCmd, args)
 	for i := 0; i < maxRetries; i++ {
 		err = nil
 		out, err := o.executer.ExecuteWithTimeout(TimeOutMultipathdCmd, multipathdCmd, args)
```

Comment:
wouldn't the `executing command` part be logged twice, even for success on first attempt?

Reply:
right, done